### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by Morelcia**
Everything should be now compliant with rules except from webp image sizes

### CODE REVIEW

**Comment by Socerest2** [comment]
Remember to compress your files!!!! arroz-chaufa-2 was too big. I compressed it for you.
[arroz-chaufa-2](https://i.ibb.co/PxLN5s0/arroz-chaufa-2.webp)

**Comment by Morelcia** [comment]
nice meme

**Comment by Socerest2** [comment]
> nice meme

huh, thats odd, it should off passed. You uploaded the file, it's under 100kb. It's erroring saying it can't find the file.

**Comment by Morelcia** [comment]
Should I git reset this compression and force push?

**Comment by Socerest2** [comment]
> nice meme

Found it, rename my file i gave you arroz_chaufa-2. I accidentally labeled it with a `-`

**Comment by Morelcia** [comment]
It's supposed to be with `-`

**Comment by LukeSmithxyz** [comment]
I don't know what you're doing with tags, but none of those make sense.

No one is going to be looking for "salt" or "pepper" or "onion" or "meat". They're not supposed to be ingredients. The tag cloud is bigger than it should be already. Keep to the meat used, the region of origin and large themes for tags.

File: src/arroz-chaufa.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Peruvian-chinese dish. Easy to cook just add and mix everything.
 
-![Arroz Chaufa](pix/arroz_chaufa-2.webp)
+![Arroz Chaufa](pix/arroz-chaufa-2.webp)
```
Comment (Socerest2) [inline src/arroz-chaufa.md:5]:
```suggestion
![Arroz Chaufa](data/pix/arroz-chaufa-2.webp)
```
Fixed Error

File: src/arroz-chaufa.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ Peruvian-chinese dish. Easy to cook just add and mix everything.
 - Soy Sauce
 - Welsh Onion
 
-![Ingredients](pix/arroz_chaufa-1.webp)
+![Ingredients](pix/arroz-chaufa-1.webp)
```
Comment (Socerest2) [inline src/arroz-chaufa.md:21]:
```suggestion
![Ingredients](data/pix/arroz-chaufa-1.webp)
```
Fixed Error

**Review by Socerest2 — CHANGES_REQUESTED** [review]
Found the check error!

**Comment by Morelcia** [comment]
> I don't know what you're doing with tags, but none of those make sense.
> 
> No one is going to be looking for "salt" or "pepper" or "onion" or "meat". They're not supposed to be ingredients. The tag cloud is bigger than it should be already. Keep to the meat used, the region of origin and large themes for tags.

I'll fix tags, should I keep "meat" tag? It's already in `brown-sauce` anyway

File: src/arroz-chaufa.md (hunk 1 above)
Comment (hallzy) [reply src/arroz-chaufa.md:5]: It is correct without the `data`. The webserver file structure is slightly different to the repository, so you don't need the `data` folder. Yes, it looks bad when you visit the markdown file here on github, but that's how it is

File: src/arroz-chaufa.md (hunk 2 above)
Comment (hallzy) [reply src/arroz-chaufa.md:21]: Same as above. `data` should not be there.

**Comment by Morelcia** [comment]
Should be better now. I tried to limit making new tags but it's just impossible to get 2 tags in something as simple as corn salsa

**Comment by hallzy** [comment]
Looks like the checks are failing on this file `data/pix/arroz_chaufa-2.webp` cause it doesn't exist... Seems like an oversight in the script. I guess because you renamed it, but the renamed file is now different because it is smaller...

We should really have a way to not check files that are deleted in the script

File: src/arroz-chaufa.md (hunk 1 above)
Comment (Socerest2) [reply src/arroz-chaufa.md:5]: > It is correct without the `data`. The webserver file structure is slightly different to the repository, so you don't need the `data` folder. Yes, it looks bad when you visit the markdown file here on github, but that's how it is

thanks!

**Review by Socerest2 — APPROVED** [review]

**Comment by Morelcia** [comment]
> Looks like the checks are failing on this file `data/pix/arroz_chaufa-2.webp` cause it doesn't exist... Seems like an oversight in the script. I guess because you renamed it, but the renamed file is now different because it is smaller...
> 
> We should really have a way to not check files that are deleted in the script

It was working correctly until I replaced old file with compressed one.
Also if tags are still bad then this task is not for my brain

**Review by hallzy — APPROVED** [review]

**Comment by LukeSmithxyz** [comment]
Get rid of meat, corn, salsa and eggs as tags. If you want corn salsa to have appropriate tags, sauce or mexican would be the obvious ones. Yeah brown sauce has/had meat but it'll be gone in a bit.

**Review by Aethusx — APPROVED** [review]

**Comment by Morelcia** [comment]
meat, corn, salsa and eggs are gone

**Pull request merged**